code/game/objects/lighting/_base_roguelight.dm

```diff
@@ -179,7 +178,19 @@
 				return
 			if(user.used_intent?.type != INTENT_SPLASH)
 				W.spark_act()
-	..()
+	. = ..()
 
 /obj/machinery/light/rogue/take_damage(damage_amount, damage_type = BRUTE, damage_flag = 0, sound_effect = 1)
-	return
+	if(!can_damage)
+		return
+	. = ..()
+
+/obj/machinery/light/rogue/break_light_tube(skip_sound_and_sparks = 0)
+	if(status == LIGHT_EMPTY || status == LIGHT_BROKEN)
+		return
```

Comment:
I'm leaving this comment here because I'm merging this and I need to remove this